index.js

```diff
@@ -1,23 +1,21 @@
 'use strict';
+
+const copyProp = (to, from, prop) =>
+  Object.defineProperty(to, prop, Object.getOwnPropertyDescriptor(from, prop));
+
 module.exports = (to, from) => {
+	copyProp(to, from, 'length');
```

Comment:
Apparently, `Object.getOwnPropertyNames(from)` returns `length`, so this is already handled by the below loop.